.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "vendor/postgres-v14"]
 	path = vendor/postgres-v14
 	url = https://github.com/neondatabase/postgres.git
-	branch = REL_14_STABLE_neon
+	branch = tristan957/v14.10
```

Comment:
Is it for testing purposes? Or just a wrong branch?

Reply:
Testing purposes. See docs/updating-postgres.md